src/time/readme.md

```diff
@@ -110,3 +110,20 @@ tick();
 ```
 
 [Try it](https://share.effector.dev/VuhhzWKE)
+
+### [Tests] Exposed timers API example
+
+```ts
+// you can attach() any source of time
```

Comment:
I suggest to rephrase a comment a bit, so it is easier to understand, what is going on - something like this

```suggestion
/**
* `timeReader` - is a either `Date.now` or the handler, that was provided via `getNow` in the configuration
*
* In your custom `readNowFx` you can change the way original time reader is used or call your own timer instead
*/
```

Maybe, it actually should just be above of the code example 🤔